validator/validator.bal

```diff
@@ -47,8 +114,10 @@ public isolated function validate(json|anydata data, typedesc<anydata>? targetFH
     anydata|constraint:Error validationResult = constraint:validate(finalData, typeDescOfData);
 
     if validationResult is constraint:Error {
-        return <r4:FHIRValidationError>r4:createFHIRError("FHIR resource validation failed",r4:ERROR, r4:INVALID, validationResult.message(),
-                                                errorType = r4:VALIDATION_ERROR, cause = validationResult);
+        log:printDebug("Error is a constraint:Error");
```

Comment:
let's add validation result as well to the debug log, and reword something like "Constraint validation error occurred"